demosdk/pom.xml

```diff
@@ -63,9 +63,9 @@
 		<jacoco.maven.plugin.version>0.8.11</jacoco.maven.plugin.version>
 
 		<!-- Kernel -->
-		<kernel.logger.logback.version>1.3.0-beta.1</kernel.logger.logback.version>
-		<kernel.core.version>1.3.0-beta.1</kernel.core.version>
-		<kerne-bom.version>1.3.0-beta.1</kerne-bom.version>
+		<kernel.logger.logback.version>1.3.0-SNAPSHOT</kernel.logger.logback.version>
+		<kernel.core.version>1.3.0-SNAPSHOT</kernel.core.version>
```

Comment:
<kernel.demographicsdk.version>1.2.0</kernel.demographicsdk.version>
it should be 1.3.0-SNAPSHOT

Reply:
corrected